deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -106,87 +104,95 @@ spec:
-            verbs:
-              - '*'
-          - apiGroups:
-              - monitoring.coreos.com
-            resources:
-              - servicemonitors
-            verbs:
-              - get
-              - create
-          - apiGroups:
-              - infinispan.org
-            resources:
-              - '*'
-            verbs:
-              - '*'
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - services/finalizers
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+              verbs:
+                - '*'
+            - apiGroups:
+                - ""
+              resources:
+                - pods/exec
+              verbs:
+                - create
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+              verbs:
+                - get
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - deployments/finalizers
+                - daemonsets
+                - replicasets
+                - statefulsets
+                - statefulsets/finalizers
+              verbs:
+                - '*'
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes
+                - routes/custom-host
+              verbs:
+                - '*'
+            - apiGroups:
+                - networking.k8s.io
+              resources:
+                - ingresses
+              verbs:
+                - '*'
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - infinispan.org
+              resources:
+                - '*'
+                - caches
+              verbs:
+                - '*'
       clusterPermissions:
         - serviceAccountName: infinispan-operator
           rules:
-          - apiGroups:
-              - ""
-            resources:
-              - nodes
-            verbs:
-              - get
-              - list
-              - watch
-          - apiGroups:
-              - apiextensions.k8s.io
-            resources:
-              - customresourcedefinitions
-              - customresourcedefinitions/status
-            verbs:
-              - get
-              - list
+            - apiGroups:
+                - ""
+              resources:
+                - nodes
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - storage.k8s.io
+              resources:
+                - storageclasses
+              verbs:
+                - get
+                - watch
```

Comment:
I guess this could fix something in the current released version

Reply:
yeah, but it's a just small case with custom storageClassName which is not so popular